packages/astro-font/utils.ts

```diff
@@ -11,25 +13,29 @@ interface Record {
 
 interface Source {
   path: string
-  css?: Record
-  style: string
+  css?: astroHTML.JSX.CSSProperties
```

Comment:
We might want to leave it as is Record so that anyone can add anything to try out even if outside set of properties. Makes it backward compatible.